### DISCUSSION TITLE
How can I echo in blade?

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [grantholle/laravel-timezone#1] [General]
Best way to echo in blade please? :)

In the other package, I just did:

`@displayDate($submission->created_at, 'M jS, Y | g:i A')`

**Comment by grantholle** [comment]
You can use normal Blade techniques to display dates using either the facade or helper function.

```blade
<p>{{ to_local_timezone($submission->created_at, 'M jS, Y | g:i A') }}</p>
```

**Reply by EriksonEnterprises** [reply]
Ahhh that makes sense 😅 I was overthinking it! Thanks
